scripts/README.md

```diff
@@ -0,0 +1,86 @@
+
+benchmark.py
+-------------------
+
+This script provides functionality for benchmarking and tracking MOOSE performance.  It can be
+used to run benchmarks, generate trend visualizations, and look at stats comparing bencharks
+between various revisions.  To run all benchmarks, just do:
+
+```
+$ ./benchmark.py --run
```

Comment:
Still not completely sold on this idea. Why do we need benchmark.py when we have the TestHarness? Note that the TestHarness doesn't nothing but find tests, create associated testers and ship them off to a Scheduler object. That's what the refactor was for.  While I'm happy that you created a new TestHarness based Tester, you still have yet another script that finds and builds Benchmark Testers which duplicates a lot of the functionality of the TestHarness. Yes, I realize that this is a simpler script but there are many reasons for that. As this tool gains in popularity, you'll be re-learning many of the lessons that we already learned during the development of the TestHarness.

Is the issue about keeping the system quiet? I'm just trying to understand the need for extra scripts.

Reply:
The script is primarily for analysis of benchmark data.  I figured there was no reason to prevent it from running benchmarks manually if someone really wanted to do that.  The functionality for running them from this script is not intended to be a part of any official workflow.  It does make it easy to benchmark any binary - even ones that aren't moose (not that we really care about that).

Reply:
And also makes it convenient for debugging the benchmarking functionality.